projects/netgrif-components-core/src/assets/i18n/de.json

```diff
@@ -211,7 +211,10 @@
             "weekend": "Das eingegebene Datum muss das Wochenende sein",
             "workday": "Das eingegebene Datum muss ein Wochentag sein",
             "enumeration": "Eine von der angebotenen Optionen muss ausgewählt sein",
-            "min": "Die eingegebene Nummer muss größer als {{length}} sein"
+            "min": "Die eingegebene Nummer muss größer als {{length}} sein",
```

Comment:
the translations are correct.

The `translationRequired` and `translationOnly` validation messages are only available in plural forms (for all supported languages), is this intended? Do we want to support singular forms in the future if only one language is required?

The required languages themselves in these messages come from "outside" as parameters. Are the names of these languages also translated?

Reply:
It's in plural form, because it refers to collection of languages regardless of that collection contains only one language.

Languages in messages are currently only in English language for each one.

Reply:
Yes, but that is sadly grammatically incorrect. I don't request that both a plural and a singular form is supported, but it marginally worsens the user experience and we should at least create a Jira issue to correct it at some point in the future.

I personally see the fact that language names are not translated as a big issue and would request introducing support at least for the languages we support directly (english, german and slovak) while at the same time giving the developers an option to define the other missing translations for other languages.

Reply:
An issue for language translations was created.